README.md

```diff
@@ -8,10 +8,19 @@ For deploying this tool to a production server, see [commcare-sync-ansible](http
 
 ## Developer Setup
 
+### Prerequisites
+- Python 3.8
+    On Ubuntu:
+    ```bash
+    sudo apt-get install python3.8 python3.8-dev
+    ```
+- Postgres
```

Comment:
I *think* sqlite / any DB works fine, and unlike HQ this project is more likely to be hosted on a different stack (though thus far we have always included our own postgres for convenience). Any reason you feel Postgres is a requirement?